docs/backup-restore-workflows.md

```diff
@@ -0,0 +1,66 @@
+# Backup Restore Workflows
+
+## Table of Contents
+
+1. [Backup Workflow on Guest Cluster](#backup-workflow-on-guest-cluster)
+2. [Restore Workflow on Guest Cluster](#restore-workflow-on-guest-cluster)
+3. [Backup Workflow on Vanilla or Supervisor Cluster](#backup-workflow-on-vanilla-or-supervisor-cluster)
+4. [Restore Workflow on Vanilla or Supervisor Cluster](#restore-workflow-on-vanilla-or-supervisor-cluster)
+
+## Backup Workflow on Guest Cluster
```

Comment:
Do we also want to have `Backup Workflow on Vanilla Cluster` here?

Reply:
3 and 4 are backup/restore workflow for both Vanilla and Supervisor.

Reply:
Let me move these workflows under sub-sections for Guest and Vanilla/Supervisor.